jekyll/_cci2/server-3-install-prerequisites.adoc

```diff
@@ -646,13 +646,24 @@ gsutil iam ch serviceAccount:<SERVICE_ACCOUNT_EMAIL>:objectAdmin gs://circleci-s
 ```
 
```

Comment:
this is the permission to add to the object storage service account in the case where workload identity is used

```shell
gcloud projects add-iam-policy-binding <PROJECT_ID> \
    --member serviceAccount:<SERVICE_ACCOUNT_EMAIL> \
    --role roles/iam.serviceAccountTokenCreator \
    --condition=None
```

Reply:
Added this command as well in the doc